MeshLib/MeshEditing/AddLayerToMesh.cpp

```diff
@@ -31,6 +32,7 @@ namespace MeshLib
 
 MeshLib::Element* extrudeElement(std::vector<MeshLib::Node*> const& subsfc_nodes,
 	MeshLib::Element const& sfc_elem,
+	MeshLib::PropertyVector<int> const& sfc_to_subsfc_id_map,
```

Comment:
The subsequent argument is a `std::map<std::size_t, std::size_t>`, why not also `size_t` here? Probably that doesn't matter so much, but maybe there should be only one index type.